doc/source/train/api/api.rst

```diff
@@ -143,6 +143,7 @@ Ray Train Utilities
     :nosignatures:
     :toctree: doc/
 
+    ~train.get_all_reported_checkpoints
```

Comment:
nit: I think you might need to update the return type to fix the formatting.

<img width="808" height="118" alt="Image" src="https://github.com/user-attachments/assets/17ea80c2-c6d2-4d28-85cd-aa95b5f95c17" />

Reply:
Changed so the entire returns docstring is on one line. 

This is what I see before my new commit - the List[ReportedCheckpoint] at the top works for me.

<img width="1482" height="814" alt="Screenshot 2025-09-02 at 8 15 01 PM" src="https://github.com/user-attachments/assets/4cea28ec-0d09-4f18-b190-f5e160a8c7fd" />